src/swf.cpp

```diff
@@ -1497,13 +1497,15 @@ void ParseThread::parseSWF(UI8 ver)
 			}
 			delete fat;
```

Comment:
I didn't remove that that line because I thought that tag pointer is different than fat pointer.

Do I understand this correctly? In line 1476 we create another pointer. Both pointers(fat, tag) points to the same memory region? So if we delete memory from one pointer the second one also is deleted?

In while loop some cases have delete tag and some don't. Is it intended? Should all of them have delete tag?

Reply:
Yes, that's exactly how it works. If you delete one pointer, the other still points to the (now invalid) memory region.

Some Tags in the while loop are needed elsewhere in the code and are deleted there.
In our case, as the FileAttributesTag is of type TAG, it's deleted in the TAG case of the switch in the while loop.

Reply:
Thank you. I was asking because I was thinking about removing "delete tag" from switch cases and moving it to the end of while loop.

I have added a patch to remove line in question. Best regards :)